test/assert.go

```diff
@@ -32,9 +32,9 @@ func assertFolderExists(t *testing.T, dir string) {
 
 func assertHasGitBranch(t *testing.T, dir, expectedBranch string) {
 	runner := NewShellRunner(dir, dir)
-	output, err := runner.Run("git", "branch")
+	outcome, err := runner.Run("git", "branch")
```

Comment:
Why is this named `outcome` instead of `result`? Appears `Result` objects are largely name `result` in functions that return them and the `activeScenarioState` also uses `lastRunResult`

Reply:
My convention is that `result` is the name of the variable containing the result of its enclosing function.
I find this helpful to navigate the code since it makes the intent and content of those variables clear.

The variable on this line contains the outcome of the call to `runner.Run`. This isn't the result of the enclosing function.I want to distinguish those two concepts.